Makefile

```diff
@@ -7,15 +7,19 @@ all: welcome
 	@exit 1
 
 doc:
+	export STRIP_FROM_INC_PATH_LIST=$(find . -type d -name include -printf " \"$PWD/%P\"")
```

Comment:
I'd suggest looking for possibilities of obtaining the directory structure that is actually under version control using git. The scripts in `dist/tools` might serve as a source of inspiration.